airbyte-integrations/connectors/destination-redshift/Dockerfile

```diff
@@ -18,3 +37,6 @@ COPY --from=build /airbyte /airbyte
 
 LABEL io.airbyte.version=0.4.7
```

Comment:
Should there be a version bump here?

Reply:
yeah, I haven't done the metadata.yaml / changelog either. just wanted to get the pr up and deal with the tedious bits later... i.e. today 😓